src/ez/DB.java

```diff
@@ -65,6 +65,23 @@
   public final String schema;
   public final boolean ssl;
 
+  public enum IsolationLevel {
+    READ_UNCOMMITTED(1),
```

Comment:
probably cleaner to reference the static fields in Connection instead of using magic numbers

Reply:
unfortunately they are defined in default scope